javascript-if/main.js

```diff
@@ -0,0 +1,108 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+// function isUnderFive(number) {
+//   if (number < 5) {
+//     return true;
+//   } else {
+//     return false;
+//   }
+// }
+
+function isUnderFive(number) {
+  return number < 5;
+}
+
+// console.log('value of isUnderFive: ', isUnderFive(4));
+// console.log('value of isUnderFive: ', isUnderFive(10));
+// console.log('value of isUnderFive: ', isUnderFive(5));
+
+function isEven(number) {
+  return number % 2 === 0;
+}
+
+// console.log('value of isEven: ', isEven(4));
+// console.log('value of isEven: ', isEven(10));
+// console.log('value of isEven: ', isEven(5));
+
+function startsWithJ(string) {
+  return string[0] === 'J';
+}
+
+// console.log('value of startsWithJ: ', startsWithJ('JavaScript'));
+// console.log('value of startsWithJ: ', startsWithJ('PHP'));
+// console.log('value of startsWithJ: ', startsWithJ('HTML'));
+// console.log('value of startsWithJ: ', startsWithJ('Java'));
+// console.log('value of startsWithJ: ', startsWithJ('Kotlin'));
+// console.log('value of startsWithJ: ', startsWithJ('C#'));
+
+function isOldEnoughToDrink(person) {
+  return person.age >= 21;
+}
+
+// var bart = {
+//   name: 'Bartholomew JoJo Simpson',
+//   age: 10
+// };
+
+// console.log('value of isOldEnoughToDrink: ', isOldEnoughToDrink(bart));
+
+function isOldEnoughToDrive(person) {
+  return person.age >= 21;
+}
```

Comment:
`21` isn't the minimum driving age:

---

![image](https://user-images.githubusercontent.com/16143771/145283128-ee57bb57-682a-45c8-b177-4a33cf7651cb.png)